setup.py

```diff
@@ -38,6 +38,13 @@
     'statsd',
 ]
 
+ENTRY_POINTS = {
+    'console_scripts': [
+        'init_schema = cliquet.scripts.init_schema:main'
```

Comment:
I believe this will create a `init_schema` command, where I think it should be prefixed (at least) with `cliquet`.